File: DAYTWO/JAVA-SCRIPT/LEVEL_ONE/TaskTen.js
const prompt = require("prompt-sync")()


let averageOfnumbers = 0;
let sumOfNumbers = 0;
let count = 0;

let index = 0;
for (;index < 3; index++){
const userInput = Number(prompt("Enter your number: "))
count += 1;
sumOfNumbers += userInput;
averageOfnumbers = sumOfNumbers / count;
}
console.log("The average is: " + averageOfnumbers);

File: DAYTWO/JAVA/LEVEL_THREE/TaskTen.java
import java.util.Scanner;

public class TaskTen{
public static void main(String... args){
Scanner input = new Scanner(System.in);

System.out.println("Enter your number");
int userInput = input.nextInt();

int count = 0;
int index = 0;
for (; index < userInput;index++){

count += 1;
index++;
}
System.out.println("The count of your numbers is: " + count);


}
}
